### ISSUE TITLE
How I can exclude some items, which I included?

### ISSUE THREAD

**Issue by felipefdl** [nfour/serverless-build-plugin#24]
Example:
```md
# on serverless.build.yml
include:
  - application/api/**
  - core/**
  - services/**
  - configuration.js

exclude:
  - "**/__test__"
  - "**/.DS_Store"
```

I want to exclude those items from inside my include, one is a folder and another is a file.

**Comment by nfour** [comment]
The project uses the `minimatch` npm module for globs, and we apply this logic for checks:

```js
        const testPattern = (pattern) => {
          return typeOf.RegExp(pattern)
              ? pattern.test(relPath)
              : glob(relPath, pattern, { dot: true });
        };

        const included = include.some(testPattern);
        const excluded = exclude.some(testPattern);

        /**
        *  When a pattern matches an exclude, it skips
        *  When a pattern doesnt match an include, it skips
        */
        if (!included || (excluded && !included)) return;
```

So it could be your exclude's aren't matching?

**Comment by felipefdl** [comment]
@nfour sorry, I don't got it. Does my logic is wrong? It should works, right?

**Comment by nfour** [comment]
@felipefdl you aren't wrong. 

```js
if (!included || (excluded && !included)) return;
```

This is exclusive logic. The second condition will never be true.
This is from when I was blacklisting instead of whitelisting.

I've just released `serverless-build-plugin@1.0.0` with the breaking change to allow excludes to actually work.
